src/renderers/webgpu/utils/WebGPUTextureUtils.js

```diff
@@ -266,7 +265,7 @@ class WebGPUTextureUtils {
 
 				console.warn( 'WebGPURenderer: Texture format not supported.' );
 
-				return this.createDefaultTexture( texture );
+				this.createDefaultTexture( texture );
```

Comment:
I think this `return` should not be deleted so the method actually returns in this block.